### PR TITLE
Transui 5 6

### PR DESCRIPTION
**Pull request by jgeofil**

### CODE REVIEW

File: src/app/components/charts/chart.directive.js [hunk 1]
```diff
@@ -5,10 +5,21 @@ angular.module('transmartBaseUi')
     return {
       templateUrl: 'app/components/charts/cohort-chart-template.html',
       link: function(scope, el, attr) {
+        //Title for the chart panel
+        scope.title = scope.label.name +
+                      " - " +
```
Comment (rnugraha) [inline src/app/components/charts/chart.directive.js]: change " to '

File: src/app/components/charts/chart.service.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ angular.module('transmartBaseUi')
              * @private
              */
             var _barChart = function (cDimension, cGroup, el, min, max, nodeTitle, width, height, btmMarg) {
+                //Determine if the number is floating point and it's precision
+                var precision = (min + "").split(".");
```
Comment (rnugraha) [inline src/app/components/charts/chart.service.js]: change " to '

File: src/app/components/charts/chart.service.js [hunk 3]
```diff
@@ -89,9 +99,13 @@ angular.module('transmartBaseUi')
             * @param charts
             */
             chartService.renderAll = function (charts) {
-                angular.forEach(charts, function (chart) {
-                  chart.render();
-                });
+              if(!charts){
+                charts = cs.charts
```
Comment (rnugraha) [inline src/app/components/charts/chart.service.js]: missing ;

File: src/app/components/charts/chart.service.js [hunk 4]
```diff
@@ -296,8 +311,38 @@ angular.module('transmartBaseUi')
...
               if(chart) {
+                var set = {
+                  rad: 0.9, // Percentage to adjust the radius of the chart
+                  legH: 0.05,// Percentage to adjust legend position in Y
+                  legW: 0, // Percentage to adjust legend position in X
+                  min: (width > height ? height : width), // Smallest of width or heigth
+                  xt: 30, // Pixels per tick in x
+                  yt: 30, // Pixels per tick in y
+                  ps: 20, // Pixels per slice for pie charts
+                }
```
Comment (rnugraha) [inline src/app/components/charts/chart.service.js]: missing ;

**Comment by jgeofil** [comment]
Done correcting use strict

**Pull request merged**